socorro/external/boto/crashstorage.py

```diff
@@ -100,9 +100,12 @@ def str_to_list(a_string):
     return json.loads(a_string)
 
 
-class BotoS3CrashStorage(CrashStorageBase):
+class BotoS3CrashStorage(SimplifiedCrashDataMixin, CrashStorageBase):
```

Comment:
This seems like a bigger architectural change with crashstorage classes, but there's no explanation of what the big picture is.

What are the main aims for this PR? What're we doing here other than generalizing "s3" to "storage"?